onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -308,78 +350,75 @@ void addGlobalMethods(py::module& m) {
 
         // default logger is needed to create the MklDNNExecutionProvider
         std::string default_logger_id{"DefaultLogger"};
-        std::unique_ptr<onnxruntime::logging::LoggingManager> default_logging_manager = 
-                  std::make_unique<LoggingManager>(
-                          std::unique_ptr<onnxruntime::logging::ISink>{ new onnxruntime::logging::CLogSink {}}, 
-                          onnxruntime::logging::Severity::kWARNING, 
-                          false,
-                          onnxruntime::logging::LoggingManager::InstanceType::Default, 
-                          &default_logger_id, 
-                          /*default_max_vlog_level*/ -1);
-     
+        std::unique_ptr<onnxruntime::logging::LoggingManager> default_logging_manager =
+            std::make_unique<LoggingManager>(
+                std::unique_ptr<onnxruntime::logging::ISink>{new onnxruntime::logging::CLogSink{}},
+                onnxruntime::logging::Severity::kWARNING,
+                false,
+                onnxruntime::logging::LoggingManager::InstanceType::Default,
+                &default_logger_id,
+                /*default_max_vlog_level*/ -1);
+
         std::vector<std::shared_ptr<onnxruntime::IExecutionProviderFactory>> factories = {
-          onnxruntime::CreateExecutionProviderFactory_CPU(0),
+            onnxruntime::CreateExecutionProviderFactory_CPU(0),
 #ifdef USE_CUDA
-          onnxruntime::CreateExecutionProviderFactory_CUDA(0),
+            onnxruntime::CreateExecutionProviderFactory_CUDA(0),
 #endif
 #ifdef USE_MKLDNN
-          onnxruntime::CreateExecutionProviderFactory_Mkldnn(1),
+            onnxruntime::CreateExecutionProviderFactory_Mkldnn(1),
 #endif
 #ifdef USE_NGRAPH
-          onnxruntime::CreateExecutionProviderFactory_NGraph("CPU"),
+            onnxruntime::CreateExecutionProviderFactory_NGraph("CPU"),
 #endif
 #ifdef USE_OPENVINO
-          onnxruntime::CreateExecutionProviderFactory_OpenVINO("CPU"),
-#endif    
-#ifdef  USE_TENSORRT    
-          onnxruntime::CreateExecutionProviderFactory_Tensorrt()
-#endif          
+            onnxruntime::CreateExecutionProviderFactory_OpenVINO("CPU"),
+#endif
+#ifdef USE_TENSORRT
+            onnxruntime::CreateExecutionProviderFactory_Tensorrt()
```

Comment:
Hi @jywu-msft , this line of code won't work. Because the CreateExecutionProviderFactory_Tensorrt function need an argument.

Reply:
Thanks. I didn't notice it because in my PR  it was just an automatic clang format change.
The issue was actually introduced in https://github.com/microsoft/onnxruntime/pull/1678
I guess it's not common to hit it because you would need to build with python enabled, TensorRT enabled and onnxruntime_PYBIND_EXPORT_OPSCHEMA